server/infrastructure/portal/mapper/PortalDataMapper.ts

```
@@ -0,0 +1,150 @@
+        code: student.mjorCd,
+        name: student.mjorNm,
+      },
+      secondaryMajor: student.the2MjorCd
+        ? {
+            code: student.the2MjorCd,
+            name: student.the2MjorNm!,
+          }
+        : undefined,
+      status: student.scrgStatNm as StudentStatusType,
+      admission: {
+        year: parseInt(student.enscYear),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)